src/containers/explore/components/EntityDataModal.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * @flow
+ */
+
+import React from 'react';
+
+import styled from 'styled-components';
+import {
+  Colors,
+  Modal,
+  ModalHeader,
+  Spinner
+} from 'lattice-ui-kit';
+import { ReduxUtils, useRequestState } from 'lattice-utils';
+import type { UUID } from 'lattice';
+import type { RequestState } from 'redux-reqseq';
+
+import EntityDataContainer from '../EntityDataContainer';
+import { ModalBody } from '../../../components';
+import { EXPLORE } from '../../../core/redux/constants';
+import {
+  EXPLORE_ENTITY_DATA,
+  EXPLORE_ENTITY_NEIGHBORS
+} from '../actions';
+
+const { NEUTRAL } = Colors;
+const { isPending, reduceRequestStates } = ReduxUtils;
+
+const StyledModalHeader = styled(ModalHeader)`
+  background: ${NEUTRAL.N600};
+  text-align: center;
+
+  h1 {
+    color: white;
+  }
+`;
```

Comment:
Figma has centered text and a white icon.

<img width="1263" alt="Screen Shot 2021-06-10 at 8 31 52 PM" src="https://user-images.githubusercontent.com/440299/121626758-e5eec600-ca2a-11eb-924b-201ecbca0d1b.png">